hamburger.js

```diff
@@ -0,0 +1,18 @@
+const hamburger = document.querySelector('.hamburger');
+const navnavMenu = document.querySelector('.navnav-menu');
+const contactLink = document.querySelector('.contact-link');
+const aboutLink = document.querySelector('.about-link');
+const portfolioLink = document.querySelector('.portfolio-link');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
```

Comment:
- [ OPTIONAL ] I think it would be great if you add some font weight to the links since it has more font weight on the original design since you should stick design as much as possible 🤩✨💯🔥